platform/ui/src/components/ViewportActionBar/ViewportActionBar.jsx

```diff
@@ -44,18 +52,30 @@ const ViewportActionBar = ({
   const renderIconStatus = () => {
     if (modality === 'SR') {
       return (
-        <div className="relative flex p-1 border rounded border-primary-light">
-          <span className="text-sm font-bold leading-none text-primary-light">
-            SR
-          </span>
-          {isLocked && (
-            <Icon
-              name="lock"
-              className="absolute w-3 text-white"
-              style={{ top: -6, right: -6 }}
-            />
+        <>
+          <div className="relative flex p-1 border rounded border-primary-light">
+            <span className="text-sm font-bold leading-none text-primary-light">
+              SR
+            </span>
+            {isLocked && (
+              <Icon
+                name="lock"
+                className="absolute w-3 text-white"
+                style={{ top: -6, right: -6 }}
+              />
+            )}
+          </div>
+          {!isLocked && !isHydrated && (
+            <div className="relative flex p-1 border rounded border-primary-light">
+              <span
+                className="text-sm font-bold leading-none text-primary-light"
+                onClick={onHydrationClick}
```

Comment:
May just want to make unlocked icon clickable? or move this to right of arrow navs

Reply:
There is no unlocked icon. Anyways, we have no scope for this from the client yet, so don't want to spend time on it until we get designs for the UI 👍 .